frontend/src/views/public/postings/survey.css

```diff
@@ -11,4 +11,12 @@ the active button to be bright. */
 
 .sv_main.sv_main .sv-boolean__switch {
     background-color: var(--primary) !important;
+}
+
+.table.sv_q_matrix thead th, .table.sv_q_matrix thead td {
+    position: sticky;
+    top: 0;
+    border-top: 0;
+    z-index: 1;
```

Comment:
This is required as the table cells below the headers are [set to be on top by default](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Positioning/Understanding_z_index/Stacking_without_z-index).